components/OneXP/OneXPBanner.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div class="one-xp-banner">
+    <div class="cds--grid">
+      <div class="one-xp-banner__container">
+        <span class="one-xp-banner__copy" v-text="copy" />
+        <UiCta
+          class="one-xp-banner__cta"
+          kind="ghost"
+          :label="cta.label"
+          :segment="cta.segment"
+          :url="cta.url"
+        />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+interface Props {
+  copy: string;
+  cta: {
+    label: string;
+    segment: CtaClickedEventProp;
+    url: string;
+  };
+}
+
+defineProps<Props>();
+</script>
+
+<style lang="scss" scoped>
```

Comment:
not sure where to leave this piece of feedback, but since the feedback pertains to the `OneXPBanner` cta, I'll leave it here.

is it possible to override the spacing between the label and the DownArrow icon? I think its currently at 1rem, but the design asks for .5rem


https://github.com/Qiskit/qiskit.org/assets/6276074/ed9b716b-22b7-42dc-a51e-cb8cddec9b05

Reply:
As discussed offline, these small design details are been discussed with the design team. It is known that our codebase won't reflect exactly the Figma design due to some structural differences, but that's OK for now.

In this case, the "ghost" UI CTA (the variant of this component) is used in many places in our pace and has that space everywhere. Changing it or introducing a new variation is not something that we can do for this specific PR but rather should be plan of a bigger restructure.